src/style.css

```diff
@@ -0,0 +1,87 @@
+@font-face {
+  font-family: "Avenir-Book";
+  src: url("./fonts/Avenir-Book.ttf");
+}
+
+@font-face {
+  font-family: "Avenir-Heavy";
+  src: url("./fonts/Avenir-Heavy.ttf");
+}
+
+.inputs {
+  box-sizing: border-box;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  width: 100%;
+  background-color: #fff;
+  font-family: "Avenir-Book", sans-serif;
+  font-weight: 300;
+  color: #3d4e61;
+}
+
+.container-bar {
+  position: relative;
+  margin-top: 20px;
+}
+
+.input-big {
+  padding: 24px 0 24px 62px;
+  height: 70px;
+  line-height: 22px;
+  font-size: 16px;
+}
+
+.input-small {
+  padding: 12px 0 12px 33px;
+  height: 42px;
+  font-size: 14px;
+  font-weight: 300;
+}
+
+.big-icon {
+  position: absolute;
+  top: 25px;
+  bottom: 26px;
+  left: 26px;
+}
+
+.small-icon {
+  position: absolute;
+  width: 11px;
+  height: 11px;
+  top: 15px;
+  left: 13px;
+}
+
+.input-big:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+}
+
+.input-big:focus {
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  border: 1px solid #e1e7ed;
+  border-radius: 3px;
+  background-color: #e5e5e5;
+  line-height: 22px;
+  font-family: "Avenir-Book", sans-serif;
+  font-weight: bold;
+  font-size: 16px;
+}
+
+.input-small:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+}
+
+.input-small:focus {
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  border-radius: 3px;
+  line-height: 19.12px;
```

Comment:
```suggestion
  line-height: 19px;
```
I kind of get it why you set `19.12px`, although it's not a common practice to use decimals with px...
round such to the nearest reasonable integer.
Pay attention that in figma design mock you have also a code `block` underneath `Typography`:
![figma](https://user-images.githubusercontent.com/39308717/158958949-1f0edd87-448f-42b2-b1e6-6c3ebf7994a4.png)